migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc

```diff
@@ -35,12 +35,12 @@ If you use a proxy server with an internally hosted replication repository, you
 
 [id="{context}_configuring-replication-repository-additional-resources"]
 [discrete]
-=== Additional resources
+=== Additional resources for configuring a replication repository
 
+* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html-single/planning_your_deployment/index#disconnected-environment_rhocs[Disconnected environment] in _Red Hat OpenShift Container Storage: Planning your deployment_.
```

Comment:
Not critical, but the way that this is worded makes me think that the book title is "Red Hat OpenShift Container Storage: Planning your deployment".

I'd consider separating the product from the book title. Could do something like:

```
in the _Planning your deployment_ guide for Red Hat OpenShift Container Storage.
```
or could simply omit the book title: 
```
in the Red Hat OpenShift Container Storage documentation.
```

Reply:
Great. I was not sure about conventions for this kind of reference.